perun-core/src/main/java/cz/metacentrum/perun/core/api/AuthzResolver.java

```diff
@@ -730,4 +764,29 @@ public static PerunPrincipal getPerunPrincipal(PerunSession sess) throws Interna
 	public static void refreshAuthz(PerunSession sess) throws InternalErrorException {
 		AuthzResolverBlImpl.refreshAuthz(sess);
 	}
+
+	/**
+	 * This methods verifies if the current principal has one of the given roles for the given object.
+	 *
+	 * @param sess session
+	 * @param complementaryObject complementary object
+	 * @param allowedRoles set of roles which are tested
+	 * @return true, if the principal is authorized, false otherwise
+	 * @throws InternalErrorException internal error
+	 */
+	public static boolean hasOneOfTheRolesForObject(
```

Comment:
That is a very unusual formatting for method header, regarding the rest of project.